src/data_types/strs.rs

```diff
@@ -347,12 +347,10 @@ impl PartialEq<CString16> for &CStr16 {
 /// [`CStr16`] can't be directly constructed. `UnalignedCStr16` instead
 /// takes a pointer to the unaligned field, which is allowed. The
 /// resulting unaligned string cannot be used directly, but must be
-/// converted to an aligned form first with [`to_cstr16`] or
-/// [`to_cstring16`].
+/// converted to an aligned form first with [`to_cstr16`].
```

Comment:
Can you say more about this change? I think the previous text seems correct.

Reply:
`.to_cstring16` doesn't exist for `UnalignedCStr16`. @nicholasbishop

Reply:
It does exist, although only if the `exts` feature is enabled since it requires allocation: https://docs.rs/uefi/latest/uefi/data_types/struct.UnalignedCStr16.html#method.to_cstring16

Reply:
Hm, I need to double check it again. I'm sure it didn't work when I ran cargo doc with all features activated.

Reply:
yeah, good catch. Sorry, thanks!